mycroft/client/speech/mic.py

```diff
@@ -199,11 +201,13 @@ class ResponsiveRecognizer(speech_recognition.Recognizer):
 
     # The maximum seconds a phrase can be recorded,
     # provided there is noise the entire time
-    RECORDING_TIMEOUT = 10.0
+    RECORDING_TIMEOUT = MIC_CONFIG.get(
+        'microphone', {}).get('recording_timeout')
```

Comment:
In python code style capital letters indicate constants, since these are no longer constants maybe we should make them lowercase object member and set them in `__init__()`?

`self.recording_timeout = config['recording_timeout']`

Reply:
moved in newer commit to class __init__ and followed code style of existing properties